src/ui/public/visualize/loader/embedded_visualize_handler.ts

```diff
@@ -148,6 +147,7 @@ export class EmbeddedVisualizeHandler {
     this.events$ = this.vis.eventsSubject.asObservable().pipe(share());
     this.events$.subscribe(event => {
       if (this.actions[event.name]) {
+        event.data.aggConfigs = this.vis.aggs.getResponseAggs();
```

Comment:
we add aggConfigs information to the event so vis_filters can correctly construct the filter.